main/ertp/guide/other-concepts.md

```diff
@@ -0,0 +1,23 @@
+# Other Concepts
+
+When looking at the code in our [tests](https://github.com/Agoric/ERTP/tree/master/test), you might see some new
+concepts:
+
+* __Vats__: All user code runs in what we call a `vat`. Within a vat,
+  code is run synchronously. Communication with another vat happens
+  asynchronously. The [SwingSet
+  infrastructure](https://github.com/Agoric/SwingSet) creates the vats
+  and makes communication between vats possible.
+
+* __E() and infix bang (!)__: Instead of `obj.foo()`, we can write
```

Comment:
We can update this to ~.

Reply:
I made some changes to this text on the ERTP github https://github.com/Agoric/ERTP/blob/master/README.md